Projects/Example Projects/ModCreatorKit/Cheats.cpp

```diff
@@ -47,10 +49,13 @@ void AddCheats()
 {
 	CheatManager.AddCheat("devContext", new ContextCheat());
 	CheatManager.AddCheat("devAnimLog", new AnimLogCheat());
+	CheatManager.AddCheat("devEffectLog", new EffectLogCheat());
 	CheatManager.AddCheat("devPackage", new PackageCheat());
 	CheatManager.AddCheat("devLogUI", new UILogCheat());
 	CheatManager.AddCheat("devInspectUI", new UIInspectCheat());
 
+	CheatManager.AddCheat("PrintCursor", new PrintCursorCheat());
```

Comment:
Maybe change the name to follow the same convention as the others, like `devPrintCursor`